packages/cli/test/commands/new.test.ts

```diff
@@ -1,5 +1,58 @@
-// import {expect, test} from '@oclif/test'
+import * as Project from '../../src/commands/new/project'
+import * as ProjectInitializer from '../../src/services/project-initializer'
+import { IConfig } from '@oclif/config'
+import * as fs from 'fs'
+import { expect } from '../expect'
+import { restore, replace, fake } from 'sinon'
+import ErrnoException = NodeJS.ErrnoException
+import { ProjectInitializerConfig } from '../../src/services/project-initializer'
 
 describe('new', (): void => {
-  // TODO: Write proper tests
+  describe('project', () => {
+    context('file generation', () => {
+      const projectName = 'test-project'
+      const projectDirectory = `./${projectName}`
+      const expectedDirectoryContent = {
+        rootPath: [
+          '.eslintignore',
+          '.eslintrc.js',
+          '.gitignore',
+          '.prettierrc.yaml',
+          'package.json',
+          'src',
+          'tsconfig.eslint.json',
+          'tsconfig.json',
+        ],
+        src: ['commands', 'events', 'event-handlers', 'entities', 'read-models', 'config', 'common', 'index.ts'],
+      }
+      const projectInitializerConfig = {
+        projectName: projectName,
+        description: '0.1.0',
+        version: '0.1.0',
+        author: 'superAuthor',
+        homepage: '',
+        license: 'MIT',
+        repository: '@boostercloud/framework-provider-aws (AWS)',
+        providerPackageName: '',
```

Comment:
While doing the test for another task I realized that this was badly configured :P
```suggestion
        repository: '',
        providerPackageName: '@boostercloud/framework-provider-aws',
```